deployment/invalidateCloudFront.js

```diff
@@ -0,0 +1,33 @@
+const aws = require('aws-sdk');
+const cloudFront = new aws.CloudFront();
+
+// This function clears the cloudFront cache for production deployments.
+// Without this function, new code changes would not be implemented until the old cache expired.
+cloudFront.listDistributions({}).promise()
```

Comment:
is there a way to query for the aliases here instead of getting everything and filtering? It's not a dealbreaker for now but down the line we might end up filtering through a pretty long list.

Reply:
You can't query for aliases. As far as I know, you can only query distributions by the distribution_id, but not by the alias. And that's a route we could go for sure. My thought was that the aliases would stay the same, but I wasn't sure how/if their distribution_ids would change.